apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -293,7 +293,7 @@ public static Task ReleaseAll()
         /// at the cost of performance due to lower reuse of sessions.
         /// This value must be less than the expire timer on the Spanner server currently set at 60 minutes.
         /// </summary>
-        public static TimeSpan PoolEvictTimeSpan { get; set; } = TimeSpan.FromMinutes(15);
+        public static TimeSpan PoolEvictTimeDelay { get; set; } = TimeSpan.FromMinutes(15);
```

Comment:
As noted before, PoolEvictionDelay would be more consistent and nicer - my guess is this was just a typo.